proposals/open-telemetry/README.adoc

```diff
@@ -1,198 +1,47 @@
 
 = MicroProfile OpenTelemetry
 
-[IMPORTANT]
-====
-The following content is moved from the original https://docs.google.com/document/d/1-mFPmFjxmOx29rDrr1qBtpOXmGZtqVquF0_BvpklO5s/edit#heading=h.q5u44wt1zwb9[Google Doc document]:
-====
+This specification is to adopt https://opentelemetry.io/ [OpenTelemetry] from CNCF to allow MicroProfile Applications benefit from 
+OpenTelemetry and enable Jakarta RESTful Web Services automatically being traced.
+
 == OpenTracing move to OpenTelemetry
+With the merge of OpenTracing and Open Census, OpenTelemetry is the new future. 
+OpenTelemetry consists of Tracing, Logging and Metrics support. This reposition is to start with Tracing part first with the potential to add logging support.
```

Comment:
May be in a later version of the README we should add plans for Metrics support (with a joined effort with MicroProfile Metrics) and explain the 4. spec Baggage (which we will support for Tracing too).

But to move forward faster, I can add a separate PR later @Emily-Jiang - we are still in the sand box :-)